### PR TITLE
Mpw/apollo studio ci

### PR DESCRIPTION
**Pull request by mpwsh**
- Publish the GraphQL server schema in apollo studio on successful release 
- Trigger the OPA workflow on hub-permissions repository to rebuild the schema and upload to S3

Merge this PR first:
https://github.com/holaplex/hub-permissions/pull/18

### CODE REVIEW

File: .github/workflows/schema_update.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+env:
+  APP_NAME: ${{ github.event.repository.name }}
+  APP_PORT: "3000"
+  APP_IMAGE_TAG: ${{ github.sha }}
+  DOCKER_NETWORK: ci
+  DATABASE_URL: "postgres://postgres:holaplex@db:5432/hub_nfts"
+  KAFKA_BROKERS: "redpanda:9092"
+  APOLLO_PROJECT: hub-dev@current
+
+jobs:
```
Comment (kespinola) [inline .github/workflows/schema_update.yml:16]: We have docker-compose that is capable of starting up services need to start up the webserver.

I found this github action or look to use`docker compose up`

File: .github/workflows/schema_update.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        run: |
+          # install rover CLI
+          curl -sSL https://rover.apollo.dev/nix/latest | sh
+          export PATH="/home/runner/.rover/bin:$PATH"
+          # Instrospect Graphql Server and push schema
+          rover subgraph introspect http://localhost:$APP_PORT/graphql | \
+          rover subgraph publish $APOLLO_PROJECT \
+            --name $APP_NAME \
+            --schema - \
+            --routing-url http://$APP_NAME/graphql
```
Comment (kespinola) [inline .github/workflows/schema_update.yml:106]: nb: A good one for a github actions

**Review by kespinola — CHANGES_REQUESTED** [review]
Can do the container setup with docker-compose of the project?

File: .github/workflows/schema_update.yml (hunk 1 above)
Comment (mpwsh) [reply .github/workflows/schema_update.yml:16]: thought about it but in this setup we are aiming for container to container connectivity, and our docker compose is meant to be used with local service -> container

Comment (kespinola) [reply .github/workflows/schema_update.yml:16]: Does it work if docker compose given a network then run this on the same network?

https://github.com/holaplex/hub-nfts/pull/46/files#diff-478656722c5288a0fec76bc23438f8ed204d922c307a1b07bf099108d58528edR64

Comment (mpwsh) [reply .github/workflows/schema_update.yml:16]: Usually yes, but the problem here is that you need to tell redpanda what the OUTSIDE hostname is on startup.
In the docker-compose file we use `localhost`, but [here](https://github.com/holaplex/hub-nfts/pull/46/files#diff-478656722c5288a0fec76bc23438f8ed204d922c307a1b07bf099108d58528edR47) we need to use the container name.
If you dont use the specified IP/HOST connection will fail.

**Pull request closed without merging**